client/app/components/packages/scope-of-work-draft/edit.hbs

```diff
@@ -0,0 +1,97 @@
+<SaveableForm 
+  @model={{@package}} 
+  @validators={{array (hash) (hash)}} 
+  as |saveableForm|
+>
+
+  <div class="grid-x grid-margin-x">
+    <div class="cell large-8">
+
+      <section class="form-section">
+        <h1 class="header-large">
+          Draft Scope of Work (SOW)
```

Comment:
Should be `(DSOW)`,  not `(SOW)`, right?

Reply:
Wireframe has (SOW), not (DSOW)